run.py

```diff
@@ -875,6 +875,49 @@ def supported_devices(
     def engine_manifest():
         return engine_manifest_loader.load_manifest()
 
+    @app.get("/setting", response_class=HTMLResponse)
+    def setting_get(request: Request):
+        settings = setting_loader.load_setting_file()
+
+        cors_policy_mode = settings.cors_policy_mode
+        allow_origin = settings.allow_origin
+
+        return setting_ui_template.TemplateResponse(
+            "ui.html",
+            {
+                "request": request,
+                "cors_policy_mode": cors_policy_mode,
+                "allow_origin": allow_origin,
+            },
+        )
+
+    @app.post("/setting", response_class=HTMLResponse)
+    def setting_post(
+        request: Request,
+        cors_policy_mode: Optional[str] = Form(None),  # noqa: B008
+        allow_origin: Optional[str] = Form(None),  # noqa: B008
+    ):
+        settings = Setting(
+            cors_policy_mode=cors_policy_mode,
+            allow_origin=allow_origin,
+        )
+
+        # None文字列が返された場合上書きをする
+        if settings.allow_origin == "None":
+            settings.allow_origin = None
```

Comment:
ちょっと自信ないのですが、空白のとき`None`の方が直感的かもと思いました！
```suggestion
        # 空白のときは上書き
        if len(settings.allow_origin) == 0:
            settings.allow_origin = None
```

Reply:
これは空白のときは何の問題もないのですが、そのまま`"None"`が渡される可能性があるためこのような書き方をしています。

Reply:
なるほどです！
この設定を利用する人がPythonのことを知っていればNoneと入力したくなるかもしれませんが、大多数の人は知らないと思います。
Pythonを知らない人にとっては、「なにも渡さない（空白を渡す）とき、なにも渡さない（Noneが渡される）」が直感的だろうなぁと思いました！

（そもそも空白のときはなにも問題ないなら、なにもしないのが一番いいかも･･･？）

Reply:
すみません説明不足でした。設定UIではあらかじめ渡されたデータがそのまま入力ボックスに入るようにしてあるので、それをそのまま保存した人向けと言った感じです。

Reply:
あ、なるほどです！！よくわかりました！
であればその`None`が入ってしまう方をなんとかしたほうが良さそうですね！

`allow_origin is None`のときは空白が渡るように`TemplateResponse`を変える、というのはどうでしょうか。